### ISSUE TITLE
Can't load OpenGL.so for module OpenGL in 1.2.5 and 1.2.6 (x86) precompiled packages 

### ISSUE THREAD

**Issue by xoan** [alexrj/Slic3r#2625]
Output for `$ ./bin/slicer --gui` in 1.2.5 and 1.2.6 in Debian Wheezy x86:

```
$ ./bin/slic3r --gui
Can't load '/home/user/Programas/Slic3r/lib/std/auto/OpenGL/OpenGL.so' for module OpenGL: /lib/i386-linux-gnu/libc.so.6: version `GLIBC_2.15' not found (required by /home/user/Programas/Slic3r/bin/../dll/libglut.so.3) at <content>/DynaLoader.pm line 157.
 at /home/user/Programas/Slic3r/lib/std/Slic3r/GUI/Plater/2DToolpaths.pm line 119
Compilation failed in require at /home/user/Programas/Slic3r/lib/std/Slic3r/GUI/Plater/2DToolpaths.pm line 119.
BEGIN failed--compilation aborted at /home/user/Programas/Slic3r/lib/std/Slic3r/GUI/Plater/2DToolpaths.pm line 119.
Compilation failed in require at /home/user/Programas/Slic3r/lib/std/Slic3r/GUI.pm line 16.
BEGIN failed--compilation aborted at /home/user/Programas/Slic3r/lib/std/Slic3r/GUI.pm line 16.
Compilation failed in require at (eval 107) line 1.
```

**Comment by xoan** [comment]
I guess the lastest experimental precompiled packages are compiled against a newer libc6 version:

* https://packages.debian.org/wheezy/libc6

Should Debian Wheezy users wait until Jessie pass to "stable"?

**Comment by lordofhyphens** [comment]
If you are using Debian, there are packages for 1.2.6 in debian
experimental. Use those or grab the source package and build your own.

**Comment by alexrj** [comment]
This is the same as #861. Packages are built on Ubuntu 14.04 LTS. I don't know how to make them compatible with older glibc...

**Comment by xoan** [comment]
Sorry for duplicating.

1.2.4-experimental works well on Debian Wheezy. Maybe libglut forces a newer symbol version in glibc, I don't know.

**Comment by lordofhyphens** [comment]
For Wheezy, I recommend building the package from the pkg source in the
debian  experimental repo

**Comment by xoan** [comment]
Fixed as in #2535

**Comment by alexrj** [comment]
Oh thank you @xoan for updating this.  So removing the GL/GLUT/whatever libraries fixed this one as well. Great!
